src/main/java/com/discount/Application.java

```diff
@@ -0,0 +1,12 @@
+package com.discount;
```

Comment:
change package structure based on pom group id and artifact id: com.exadel.discount